src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,13 +1,30 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Excavator
- * Do not remove no-args constructor
- */
 public class Excavator extends Machine {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line before the class declaration.